app/models/notification_blueprint.rb

```diff
@@ -10,7 +10,6 @@ class NotificationBlueprint < ActiveRecord::Base
   has_many :notifications, :dependent => :destroy
   store :actions, :accessors => [:links], :coder => JSON
 
-  validates :message, :presence => true
```

Comment:
The message is created completely from scratch on every notification, e.g: the summary of the blog entry, that does not have to rely on any subject.